cli/remote_build.go

```diff
@@ -780,9 +782,15 @@ func (cmd *RemoteCmd) submitBuild(project *Project, tagMap map[string]string) er
 	case 401:
 		submitErrored()
 		return fmt.Errorf("Unauthorized, authentication failed.\nPlease `yb login` again.")
-	case 400:
+	case 403:
 		submitErrored()
-		return fmt.Errorf("Invalid data sent to the YB API")
+		if cmd.publicRepo {
+			return fmt.Errorf("This should not happen, please open a support inquery with YB")
+		} else {
+			return fmt.Errorf("Tried to build a private repository of a organization of which you're not part of.\nPlease check with the '%s' admins", project.OrgSlug)
+		}
+	case 412:
+		// TODO Show helpfull message with App URL to fix GH App installation issue
```

Comment:
@amyhua I need your help with this.

Reply:
it'd be nice if we returned error type enums (versus specific messages) when throwing errors, so the frontend can iterate on the messaging as needed for the user experience. E.g.
ErrorType.ProviderAppNotInstalled,
ErrorType.ProviderAppInstallRepoNotSelected,
ErrorType.OrgMembershipMissing,
ErrorType.PublicReposNotSupported